qt/scientific_interfaces/ISISReflectometry/test/Options/OptionsDialogPresenterTest.h

```diff
@@ -195,5 +187,7 @@ class OptionsDialogPresenterTest : public CxxTest::TestSuite {
     TS_ASSERT(Mock::VerifyAndClearExpectations(&m_view));
     TS_ASSERT(Mock::VerifyAndClearExpectations(m_model));
     TS_ASSERT(Mock::VerifyAndClearExpectations(&mainWindowSubscriber));
```

Comment:
I think we can remove `mainWindowSubscriber` as a parameter from the `verifyAndClear` method, and remove this line that clears the expectations for that variable, so that the method would become:

```
  void verifyAndClear() {
    TS_ASSERT(Mock::VerifyAndClearExpectations(&m_view));
    TS_ASSERT(Mock::VerifyAndClearExpectations(m_model));
    TS_ASSERT(Mock::VerifyAndClearExpectations(&m_modelUnsuccessfulDefaults));
    TS_ASSERT(Mock::VerifyAndClearExpectations(&m_modelUnsuccessfulLoad));
```

Reply:
I have adjusted the PR to remove the mainWindowSubcriber as a parameter from the verifyAndClear method.
I also made modification to link the issue to the PR using `Closes`